stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/LinkAttachmentView.kt

```diff
@@ -46,9 +46,29 @@ internal class LinkAttachmentView : FrameLayout {
         defStyleAttr
     )
 
+    /**
+     * Displays the given attachment.
+     *
+     * @param attachment The attachment to be displayed.
+     * @param style The style used for applying various things such as text styles.
+     */
     fun showLinkAttachment(attachment: Attachment, style: MessageListItemStyle) {
-        previewUrl = attachment.titleLink ?: attachment.ogUrl
+        previewUrl = attachment.thumbUrl
+        showTitle(attachment, style)
+        showDescription(attachment, style)
```

Comment:
Just an attempt to break this large function into smaller more readable pieces.

Reply:
Looks really good and on the same level of abstraction